src/main/java/org/eclipse/golo/runtime/PropertyMethodFinder.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2012-2015 Institut National des Sciences Appliquées de Lyon (INSA-Lyon)
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.golo.runtime;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+import static java.lang.invoke.MethodHandles.*;
+import static java.lang.invoke.MethodType.methodType;
+import static java.util.Arrays.copyOfRange;
+
+public class PropertyMethodFinder extends RegularMethodFinder {
```

Comment:
I use `RegularMethodFinder#findInMethods` so I make the method protected and I extended it.
If don't see objections to make it public I can, indeed, extend `MethodFinder` and delegate the property method research to `RMF`